sdcm/cluster_k8s/__init__.py

```diff
@@ -616,19 +622,25 @@ def upgrade_scylla_cluster(self, new_version: str) -> None:
         self.replace_scylla_cluster_value("/spec/version", new_version)
         self.rollout_restart()
 
-    def update_scylla_config(self) -> bool:
+    def update_scylla_config(self):
         with self.scylla_yaml_lock:
-            if not self.scylla_yaml_update_required:
-                return False
+            scylla_yaml = self.scylla_yaml.copy()
+            # Remove parameters not allowed by scylla-operator
+            for param in ['seed_provider', 'rpc_address', 'cluster_name', 'listen_address']:
+                if param in scylla_yaml:
+                    del scylla_yaml[param]
```

Comment:
Is it a real issue? I don't understand the code path how it can be found in `scylla_yaml`?

Btw, you don't use your copy below.

Reply:
removed